### PR TITLE
close #44 translate and update footer

### PR DESCRIPTION
**Pull request by francisco-core**
- add i18n translation to the footer
- update text to reflect the fact that we are no longer an
informal organization.

close https://github.com/PrivacyLx/privacylx-issue-tracker/issues/44

See if you like the text. Maybe it needs changing

### CODE REVIEW

File: themes/hugo-bootstrap/i18n/en.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@
 - id: translations
   translation: "Translations"
 
+- id: footer-text
+  translation: "We are a non-profit based in Lisbon that aims to raise awareness on privacy issues"
```
Comment (anadahz) [inline themes/hugo-bootstrap/i18n/en.yaml]: Is non-profit the correct term ot is it better to mention non profit association?

Comment (francisco-core) [reply themes/hugo-bootstrap/i18n/en.yaml]: would association be better? It's just that "non-profit association" is too long

Comment (anadahz) [reply themes/hugo-bootstrap/i18n/en.yaml]: It's quite important to describe what PrivacyLx is really is. IMHO non-profit isn't the same as an non-profit association. There may be a more precise word for it.

Comment (francisco-core) [reply themes/hugo-bootstrap/i18n/en.yaml]: Then we can just put association, I guess. But it doesn't sound as good. Also, there are no other kinds of non-profits in Portugal.

Comment (francisco-core) [reply themes/hugo-bootstrap/i18n/en.yaml]: it should be fine now

**Comment by anadahz** [comment]
Thanks, looks good, merging.

**Pull request merged**